README.md

```diff
@@ -43,13 +43,13 @@ Further details of usage of the bootloader can be found in the [Firmware options
 
 The Factory Image can be flashed to a module to restore it to the factory state. Note that erasing the device will result in loss of any settings. These should be stored for restoration following erasing of the module followed by flashing of the Factory Image.
 
-[RM126x product brief]: <https://www.lairdconnect.com/documentation/product-brief-rm126x-series>
-[RM126x module datasheet]: <https://www.lairdconnect.com/documentation/datasheet-rm126x-lorawan-module>
-[RM126x AT Interface guide]: <https://www.lairdconnect.com/documentation/user-guide-rm126x-at-interface-application>
-[RM126x AT Interface release notes]: <https://www.lairdconnect.com/documentation/release-notes-rm126x-series>
-[RM126x DVK user guide]: <https://www.lairdconnect.com/documentation/user-guide-rm126x-development-kit>
-[Native C development guide]: <https://www.lairdconnect.com/documentation/user-guide-lyra-series-c-code-development>
-[Firmware options and upgrade guide]: <https://www.lairdconnect.com/documentation/user-guide-firmware-options-and-upgrading-rm126x-series>
-[RM126x Releases Page]: <https://github.com/LairdCP/RM126x_Firmware/releases/tag/GA1.3>
+[RM126x product brief]: <https://www.ezurio.com/documentation/product-brief-rm126x-series>
+[RM126x module datasheet]: <https://www.ezurio.com/documentation/datasheet-rm126x-lorawan-module>
+[RM126x AT Interface guide]: <https://www.ezurio.com/documentation/user-guide-rm126x-at-interface-application>
+[RM126x AT Interface release notes]: <https://www.ezurio.com/documentation/release-notes-rm126x-series>
```

Comment:
This one opens the BL5340PA datasheet - maybe could check with Dave?

Reply:
Dave Has fixed the link so should be all okay now.